### ISSUE TITLE
ArgumentError raised in Ruby 3.0

### ISSUE THREAD

**Issue by patorash** [yuyasat/rspec-retry_ex#6]
I was using rspec-retry_ex and got an ArgumentError in Ruby 3.0

```
Failure/Error:
  retry_ex(count: 3, after_retry: -> { sleep 0.5 }) do
    fill_in 'name', with: 'Tom'
    find('.rspec-btn-********').click
  end

ArgumentError:
  wrong number of arguments (given 1, expected 0)
```

It need to support for keyword arguments.

**Comment by patorash** [comment]
I confirmed that it was fixed in #5 . Thanks!

**Comment by yuyasat** [comment]
@patorash Thank you so much!
